script.js

```diff
@@ -0,0 +1,221 @@
+// User starts with either X or O
+// User clicks on a space, and that space shows an X or an O
+  // space displays the X or O, depending on which one was currently in effect
+  // turn switches to other player
+// Play continues until one of two things happens:
+    // No spaces remain
+    // A player, either X or O, has won
+// Reset button at the bottom of the page clears all entries
+
+var squareOne = $("#topRow .left");
+var squareTwo = $("#topRow .middle");
+var squareThree = $("#topRow .right");
+var squareFour = $("#middleRow .left");
+var squareFive = $("#middleRow .middle");
+var squareSix = $("#middleRow .right");
+var squareSeven = $("#bottomRow .left");
+var squareEight = $("#bottomRow .middle");
+var squareNine = $("#bottomRow .right");
+var resetButton = $("button");
+
+var playerTwo = false;
+
+function getWinner () {
+if (((squareOne.html() === "X") && (squareTwo.html() === "X") && (squareThree.html() === "X")) ||
+    ((squareFour.html() === "X") && (squareFive.html() === "X") && (squareSix.html() === "X"))  ||
+    ((squareSeven.html() === "X") && (squareEight.html() === "X") && (squareNine.html() === "X"))  ||
+    ((squareOne.html() === "X") && (squareFour.html() === "X") && (squareSeven.html() === "X"))  ||
+    ((squareTwo.html() === "X") && (squareFive.html() === "X") && (squareEight.html() === "X"))  ||
+    ((squareThree.html() === "X") && (squareSix.html() === "X") && (squareNine.html() === "X"))  ||
+    ((squareOne.html() === "X") && (squareFive.html() === "X") && (squareNine.html() === "X"))  ||
+    ((squareThree.html() === "X") && (squareFive.html() === "X") && (squareSeven.html() === "X"))
+) { alertify.alert("X has triumped!");
+} else if (((squareOne.html() === "O") && (squareTwo.html() === "O") && (squareThree.html() === "O")) ||
+    ((squareFour.html() === "O") && (squareFive.html() === "O") && (squareSix.html() === "O"))  ||
+    ((squareSeven.html() === "O") && (squareEight.html() === "O") && (squareNine.html() === "O"))  ||
+    ((squareOne.html() === "O") && (squareFour.html() === "O") && (squareSeven.html() === "O"))  ||
+    ((squareTwo.html() === "O") && (squareFive.html() === "O") && (squareEight.html() === "O"))  ||
+    ((squareThree.html() === "O") && (squareSix.html() === "O") && (squareNine.html() === "O"))  ||
+    ((squareOne.html() === "O") && (squareFive.html() === "O") && (squareNine.html() === "O"))  ||
+    ((squareThree.html() === "O") && (squareFive.html() === "O") && (squareSeven.html() === "O"))
+  ) { alertify.alert("O stands victorious!");
+  }
+}
```

Comment:
Concerning the above comment. What if each box had indexes that represented each of the boxes, like this:

1 2 3
4 5 6
7 8 9

Then we could say some winning combinations would be `[1, 2, 3]`  or `[1, 5, 9]`. Then we could have an array of all the winning combinations and loops through those data structures to find out the winner.